packages/std/Cargo.toml

```diff
@@ -62,6 +62,7 @@ serde-json-wasm = { version = "0.5.0" }
 thiserror = "1.0.26"
 bnum = "0.8.0"
 static_assertions = "1.1.0"
+bech32 = "0.9.1"
```

Comment:
You can move that down to the block where `cosmwasm-crypto` is as well since this is only needed for the MockApi implementation

Reply:
Done.